Makefile

```diff
@@ -481,11 +481,12 @@ endef
 # Allow specifying just the subproject, in the keyboard directory, which will compile all keymaps
 SUBPROJECTS := $(notdir $(patsubst %/Makefile,%,$(wildcard ./*/Makefile)))
 .PHONY: $(SUBPROJECTS)
-$(SUBPROJECTS): %: %-allkm 
+$(SUBPROJECTS): %: %-allkm
 
 # Let's match everything, we handle all the rule parsing ourselves
 .PHONY: %
-%: 
+ifndef SKIP_GIT
```

Comment:
Since you cannot use `ifdef`s inside rules, I would extract the submodule check to it's own mutliline variable, using `define` and `endef`. Inside that you can use `ifdef`, but if you want to indent the code you need to use spaces, not tabs. There should already be a few examples of this in the makefile

To call the multiline varable, you need to use the `$(eval $(call VARIABLE_NAME))` syntax.